.github/workflows/build-python.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
           pipenv install --dev
           pipenv lock --keep-outdated > pipenv-requirements.txt
           pipenv requirements > requirements.txt
-          pipenv run pytest tests --junitxml=report.xml
+          pipenv run tests --junitxml=report.xml
```

Comment:
@hawk-sergey-kurilo about this I have no knowledge at all. Other repositories using this workflow are ai-data-sink and entity-resolution. If you are sure that this change is ok for all of them we can merge it

Reply:
this is ok